core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -403,6 +404,11 @@ public SystemSessionProperties(
                         "Maximum scan physical bytes of a query",
                         queryManagerConfig.getQueryMaxScanPhysicalBytes().orElse(null),
                         false),
+                dataSizeProperty(
```

Comment:
There should be also property in `memoryManagerConfig` similar to `QUERY_MAX_SCAN_PHYSICAL_BYTES `